modules/nf-core/nextdenovo/meta.yml

```diff
@@ -0,0 +1,70 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "nextdenovo"
+description: string graph-based de novo assembler for long reads (CLR, HiFi and ONT)
+keywords:
+  - assembler
+  - long reads
+  - correaction
+  - genomics
+tools:
+  - "nextdenovo":
+      description: "String graph-based de novo assembler for long reads (CLR, HiFi and ONT)"
+      homepage: "https://nextdenovo.readthedocs.io/en/latest/index.html"
+      documentation: "https://nextdenovo.readthedocs.io/en/latest/OPTION.html"
+      tool_dev_url: "https://github.com/Nextomics/NextDenovo"
+      doi: "10.1101/2023.03.09.531669"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: long reads file (fastq)
+      pattern: "*.{fastq.gz}"
+  - read_type:
+      type: string
+      description: type of long reads used as input
+      pattern: "{clr,ont,hifi}"
+  - step:
+      type: string
+      description: task need to run, correct = only do the correction step, assemble = only do the assembly step, all = correct + assemble.
+      pattern: "{all, correct, assemble}"
+  - input_type:
+      type: string
+      description: input reads type
+      pattern: "{raw, corrected}"
+  - genome_size:
+      type: string
+      description: estimated genome size, suffix K/M/G recognized
+      pattern: "{3g}"
```

Comment:
What is this pattern actually matching?

Reply:
It is an example of what it should look like for a genome of 3Gb. Should it be done differently?